kogito-bom/pom.xml

```diff
@@ -53,2739 +53,1251 @@
         <classifier>sources</classifier>
       </dependency>
-
       <dependency>
         <groupId>org.kie.kogito</groupId>
-        <artifactId>kogito-addons-quarkus-messaging-common</artifactId>
+        <artifactId>kogito-addons-quarkus-source-files-deployment</artifactId>
         <version>${project.version}</version>
       </dependency>
       <dependency>
         <groupId>org.kie.kogito</groupId>
-        <artifactId>kogito-addons-quarkus-messaging-common</artifactId>
+        <artifactId>kogito-addons-quarkus-source-files-deployment</artifactId>
         <version>${project.version}</version>
         <classifier>sources</classifier>
       </dependency>
       <dependency>
         <groupId>org.kie.kogito</groupId>
-        <artifactId>kogito-addons-springboot-messaging</artifactId>
+        <artifactId>kogito-addons-rest-exception-handler</artifactId>
         <version>${project.version}</version>
       </dependency>
+
       <dependency>
         <groupId>org.kie.kogito</groupId>
-        <artifactId>kogito-addons-springboot-messaging</artifactId>
+        <artifactId>kogito-addons-rest-exception-handler</artifactId>
         <version>${project.version}</version>
         <classifier>sources</classifier>
       </dependency>
-
       <dependency>
         <groupId>org.kie.kogito</groupId>
-        <artifactId>kogito-timer</artifactId>
+        <artifactId>kogito-addons-quarkus-rest-exception-handler</artifactId>
         <version>${project.version}</version>
       </dependency>
+
       <dependency>
         <groupId>org.kie.kogito</groupId>
-        <artifactId>kogito-timer</artifactId>
+        <artifactId>kogito-addons-quarkus-rest-exception-handler</artifactId>
         <version>${project.version}</version>
         <classifier>sources</classifier>
       </dependency>
-
       <dependency>
         <groupId>org.kie.kogito</groupId>
-        <artifactId>kogito-maven-plugin</artifactId>
+        <artifactId>kogito-addons-process-svg</artifactId>
         <version>${project.version}</version>
       </dependency>
+
+      <!-- Codegen modules -->
       <dependency>
         <groupId>org.kie.kogito</groupId>
-        <artifactId>kogito-maven-plugin</artifactId>
+        <artifactId>kogito-addons-process-svg</artifactId>
         <version>${project.version}</version>
         <classifier>sources</classifier>
       </dependency>
-
-      <!-- Codegen modules -->
       <dependency>
         <groupId>org.kie.kogito</groupId>
-        <artifactId>kogito-codegen-api</artifactId>
+        <artifactId>kogito-addons-process-svg</artifactId>
         <version>${project.version}</version>
+        <type>test-jar</type>
       </dependency>
       <dependency>
         <groupId>org.kie.kogito</groupId>
-        <artifactId>kogito-codegen-api</artifactId>
+        <artifactId>kogito-addons-jobs-management-common</artifactId>
+        <version>${project.version}</version>
+        <type>jar</type>
```

Comment:
Same here

Reply:
<type>jar</type> not needed, I will remove it